README.md

```diff
@@ -76,3 +76,48 @@ Solution:
 3. Change related ip of host.docker.internal and gateway.docker.internal to your new ip (found in terminal using the ipconfig command: e.g. 192.168.0.1)
 4. save
 5. restart the application.
+
+## Adding an ADR Algorithm
```

Comment:
Thank you for the pre-built module! For security reasons, we will rebuild it with what's given and update it after the PR.

Reply:
You are right about that. 
Do you want me to exclude the one that I build for the PR?

Reply:
Good point. You're welcome to do that.